pyTANSPEC/config/spectrum_extractor_TANSPEC.config

```diff
@@ -76,11 +76,11 @@ RectificationWindow = (-8,8)
 # Extraction Methods: Sum, FlatRelativeOptimal, Optimal
 ExtractionMethod = Sum
 #Sum extraction window is the aperture for sum extraction
-#ApertureWindow = (-8,8)
+ApertureWindow = (-8,8)
```

Comment:
Bug ALERT: This is overriding the user defined aperture. Shouldn't this be kept empty?